### PR TITLE
update dbt commands

### PR DESCRIPTION
**Pull request by mirnawong1**
updating dbt commands so they're in parity for ide v1.2 backend changes

Launching Aug 3rd to coincide with ide v1.2 cc @matthewshaver

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ sidebar_label: Develop in the IDE
 tags: [IDE]
 ---
 
-The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly. The IDE leverages the open-source [dbt-rpc](/reference/commands/rpc) plugin to recompile only the changes made in your project.
-
+The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly. The IDE leverages the dbt-server, making the IDE faster and more reliable.
```
Comment (davidharting) [inline website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md]: The second sentence here may be worth removing or amending.

We removed dbt-server from IDE v1.2 and instead are using a closed-source fork called `ide-server`.
We also are not yet able to claim that it is faster or more reliable than local development 😓 Although that is certainly a goal!

**Review by davidharting — COMMENTED** [review]
Left one note to discuss before approving, but everything else is looking great thanks!

File: website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md]:
```suggestion
The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly.

**Comment by mirnawong1** [comment]
thanks so much @davidharting ! i've removed that 2nd sentence now 🙏

**Review by davidharting — APPROVED** [review]
Thanks Mirna!

**Review by matthewshaver — APPROVED** [review]

**Comment by matthewshaver** [comment]
Holding in draft until conversation with @jtcohen6

**Review by davidharting — APPROVED** [review]
Went back for another review - still looks great! 🚢

**Pull request merged**